neqo-common/src/hrtime.rs

```diff
@@ -390,7 +390,7 @@ mod test {
             let e = Instant::now();
             let actual = e - s;
             let lag = actual - d;
-            println!("sleep({:?}) \u{2192} {:?} \u{394}{:?}", d, actual, lag);
+            println!("sleep({d:?}) \u{2192} {actual:?} \u{394}{lag:?}");
```

Comment:
I'm surprised. I thought this was new to 1.66...

Reply:
Ah no, the clippy warning is new to 1.66, format strings capturing local variables is new to 1.58.